deps.edn

```diff
@@ -10,24 +10,24 @@
   org.clj-commons/dirigiste {:mvn/version "1.0.4"},
```

Comment:
That good old `deps.edn` that needs to be updated.

Reply:
Yup, already done - the CI job reminded me :smile: